File: articles/the-unknown-the-variable-and-islamic-mathematical-philosophy/java/SquareCompletion.java
public class SquareCompletion {
    public static void main(String[] args) {
        double b = 10.0;
        double c = 39.0;
        double completion = Math.pow(b / 2.0, 2);
        System.out.printf("completion_term=%.6f%n", completion);
        System.out.printf("completed_rhs=%.6f%n", c + completion);
    }
}

File: articles/the-unknown-the-variable-and-islamic-mathematical-philosophy/go/square_completion.go
package main

import "fmt"

func main() {
	b := 10.0
	c := 39.0
	completion := (b / 2.0) * (b / 2.0)
	fmt.Printf("completion_term=%.6f\n", completion)
	fmt.Printf("completed_rhs=%.6f\n", c+completion)
}
